libs/common/src/utils/monetary-calculation.ts

```diff
@@ -0,0 +1,23 @@
+import { Big } from 'big.js';
+
+/**
+ * Applies an exchange rate to a monetary amount using the big.js library
+ * @param amount - The monetary amount to apply the exchange rate to
+ * @param exchangeRate - The exchange rate to apply to `amount`
+ * @param operation - Whether to multiply or divide the `amount` by the `exchangeRate`
+ * @param [precision=2] - The number of decimal places of precision to use when applying the exchange rate
+ * @returns The amount after the exchange rate has been applied
+ */
+export const applyExchangeRateToAmount = (amount: number, exchangeRate: number, operation: 'multiply' | 'divide', precision = 2): number => {
```

Comment:
Seeing as we are only using the divide one at the minute, I have just replaced this with a single `divideBy` helper. We can add a multiply one in future if we need :)

Reply:
I still think it is slightly hard for the less mathematically inclined to follow _why_ we divide. I have a maths degree and still it confuses me, I actually liked `applyExchangeRateToAmount` because you didn't need to know what maths it was doing (except it was in the parameters hence the comment). I guess the crucial information here is which way the exchange is going, so it would be good to capture that somehow, perhaps with a parameter names along the lines of `baseCurrencyAmount` and `baseCurrencyToTargetCurrencyExchangeRate`, although perhaps dividing is the other way, 'targetToBase'(?) but I want the function to do that thinking for me 😅 Although I guess `paymentExchangeRate` actually doesn't tell us which way it goes, I'm assuming from what you have written it is  payment currency to fee currency?

Reply:
I've added a method to the `FeeRecordEntity` class now rather than having these new helpers, so the way we convert between currencies is now specific to the entity

Reply:
Yep I like this, think we should stick with this pattern for doing conversions between currencies where it's not obvious what the conversion is representing, nice